src/lib/cfa/nm_connection.rb

```diff
@@ -31,15 +35,26 @@ class NmConnection < BaseModel
     KNOWN_SECTIONS = [
       "bridge", "connection", "ethernet", "ipv4", "ipv6", "vlan", "wifi", "wifi_security"
     ].freeze
+    SYSTEM_CONNECTIONS_DIR = "/etc/NetworkManager/system-connections".freeze
 
+    class << self
+      # Returns all connection definition files
+      #
+      # @return [Array<NmConnection>]
+      def all
+        directory = File.join(Yast::Installation.destdir, SYSTEM_CONNECTIONS_DIR)
```

Comment:
Just wondering if this could be a problem during a live installation and if it would makes sense to use  the Yast::WFM.scr_root as it is defined by the target file https://github.com/yast/yast-yast2/blob/1629c3a520ebd8e8b93f1161f06d9e140d65b9ec/library/system/src/lib/yast2/target_file.rb#L43.

OR even use the `default_file_handler.final_path` method